server/src/utils/fight/fightMethods.ts

```diff
@@ -945,26 +957,31 @@ const activateSuper = (
             a: StepType.Leave,
             f: pet.index,
           });
-
           // Remove pet from fight
           fightData.fighters = fightData.fighters
             .filter((f) => f.index !== pet.index);
+        } else {
+          // Set remaining pets to play just after
+          pet.initiative = fighter.initiative - 0.01;
+          unafraidPetIndexes.push(pet.index);
         }
       }
 
-      // Abort if no pet feared
-      if (fearSteps.length === 0) return false;
+      // Increase opponents initiative
+      getOpponents({ fightData, fighter, bruteOnly: true })
+        .forEach((opponent) => opponent.initiative += 0.2 );
```

Comment:
forEach should never return any value

```suggestion
        .forEach((opponent) => {
       	  opponent.initiative += 0.2;
        });
```